charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -38,3 +38,7 @@ keywords:
   - infrastructure
   - newrelic
   - monitoring
+
+annotations:
+  artifacthub.io/changes: |
+    - Added liveness probe
```

Comment:
NIT: adding this now will make the future changes even more painful.

I would prefer to make the effort in automating the release process than add more complexity to the already complex process.